activerecord/lib/active_record/schema_dumper.rb

```diff
@@ -71,11 +71,9 @@ def header(stream)
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# Note that this schema.rb definition is the authoritative source for your
-# database schema. If you need to create the application database on another
-# system, you should be using db:schema:load, not running all the migrations
-# from scratch. The latter is a flawed and unsustainable approach (the more migrations
-# you'll amass, the slower it'll run and the greater likelihood for issues).
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` may be
+# faster than running all migrations from scratch.
```

Comment:
"may be faster" is not what we want to pass here. You should not run the migrations at all. So what we want is stronger than this.